src/selectors/petitionsGroups.js

```diff
@@ -1,12 +1,11 @@
 import { set } from 'lodash/object';
-import { forEach } from 'lodash/collection';
 import getPetitions from 'selectors/petitions';
 
-export default (petitions = {}, groups = []) => {
+export default (petitions, groups) => {
   const petitionGroups = {};
 
-  forEach(groups, (group) =>
-    set(petitionGroups, group, getPetitions(petitions[group].data || []))
+  groups.forEach((group) =>
+    set(petitionGroups, group.group, getPetitions(petitions[group.group] && petitions[group.group].data || []))
```

Comment:
Quite irrelevant but not a fan of the `group.group` naming. But don't change it if there's nothing better :)